packages/shared/src/components/Feed.spec.tsx

```diff
@@ -223,6 +224,9 @@ it('should add one placeholder when loading', async () => {
 });
 
 it('should replace placeholders with posts and ad', async () => {
+  // is desktop
+  jest.spyOn(hooks, 'useViewSize').mockImplementation(() => true);
```

Comment:
Can we add this mock to a beforeEach or similar? Seems wasteful to repeat in every test.